SV-611 search result page approved for standard version

ben1stone opened the pull request.

CodeWolf86 commented on src/SFA.DAS.AssessorService.Application/Handlers/Search/SearchHandler.cs:
```diff
@@ -111,10 +110,12 @@ private async Task<List<SearchResult>> Search(SearchQuery request, CancellationT
 
         private async Task<List<int>> GetEpaoStandards(Organisation thisEpao)
```
Can we rename the method? `GetApprovedStandardsForEpao`? `GetEpaoApprovedStandardsWithAtLeastOneVersion` ? I'm trying to make it more descriptive to say what it's doing, GetEpaoStandards does say what it's doing but not the whole story.

ben1stone commented:
@alandburns I've updated to account for the scenario you described on the Jira ticket comment. I've added if's in the view around the certificate data fields because these are filled during the mapping if it is the correct EPAO so if it is a different EPAO they will be null
